fineract-provider/src/main/java/org/apache/fineract/organisation/teller/service/TellerManagementReadPlatformServiceImpl.java

```diff
@@ -261,8 +261,10 @@ private String getTellerCriteria(final String sqlSearch, final Long officeId, fi
             extraCriteria.append(" and status = 300 ");
         } else if (status.equalsIgnoreCase("inActive")) {
             extraCriteria.append(" and status = 0 ");
-        } else if (status.equalsIgnoreCase("all")) {} else {
+        } else {
+            if (!status.equalsIgnoreCase("all")){
```

Comment:
Why changing this? Just curious..

Reply:
Else if block is empty for condition (status.equalsIgnoreCase("all")), this is not supported.
But as else if block works, we cannot simply remove it, or even if the (status.equalsIgnoreCase("all")) value is true it will go in else.

We don't want that, so I have added a extra check in else block to make sure (status.equalsIgnoreCase("all")) is not true, and only then perform else action that what we received is incorrect and act on it.

Reply:
got it, thanks for clarifying!